proto/celestia/da/data_availability_header.proto

```diff
@@ -1,7 +1,7 @@
 syntax = "proto3";
-package da;
+package celestia.da;
 
-option go_package = "github.com/celestiaorg/celestia-app/proto/da";
+option go_package = "github.com/celestiaorg/celestia-app/proto/celestia/da";
```

Comment:
[Question]
Why did we add the `/celestia/` to the `go_package` for DA and to for others?
IMO, the `celestia` is a bit redundant in the `da` package as we're already in the Celestia repo:
```
daproto "github.com/celestiaorg/celestia-app/proto/celestia/da"
```

Reply:
Because in the other packages we generate to the types directory but here we just generate the go code to the same directory